rectools/visuals/metrics_app.py

```diff
@@ -0,0 +1,128 @@
+import typing as tp
+
+import ipywidgets as widgets
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+from IPython.display import display
+
+from rectools import Columns
+
+
+class MetricsApp:
+    """
+    Jupyter widgets app for metric visualization and comparison.
+    Do not create instances of this class directly. Use `MetricsApp.construct` method instead.
+    """
+
+    def __init__(
+        self,
+        df_metrics_data: pd.DataFrame,
+        chart_width: tp.Union[float, int] = 800,
```

Comment:
1. How it can be float?
2. Plotly allows you to pass `None`, so it's better to use it here as well
3. In order not to list here all the plotly params and in the same time give the user maximum flexibility we could simply make a param `plotly_kwargs: Dict[str, Any]`

Reply:
I found that in Plotly's source code. However, the suggestion to switch to kwargs was great, so I've implemented it.